src/main/kotlin/com/cognifide/gradle/aem/instance/InstanceSync.kt

```diff
@@ -1,25 +1,34 @@
 package com.cognifide.gradle.aem.instance
 
 import com.cognifide.gradle.aem.base.api.AemConfig
+import com.cognifide.gradle.aem.internal.Behaviors
 import com.cognifide.gradle.aem.internal.Patterns
 import com.cognifide.gradle.aem.pkg.PackagePlugin
 import com.cognifide.gradle.aem.pkg.deploy.*
-import org.apache.commons.httpclient.HttpClient
-import org.apache.commons.httpclient.HttpMethod
-import org.apache.commons.httpclient.HttpStatus
-import org.apache.commons.httpclient.UsernamePasswordCredentials
-import org.apache.commons.httpclient.auth.AuthScope
-import org.apache.commons.httpclient.methods.GetMethod
-import org.apache.commons.httpclient.methods.PostMethod
-import org.apache.commons.httpclient.methods.multipart.*
-import org.apache.commons.httpclient.params.HttpConnectionParams
 import org.apache.commons.io.IOUtils
+import org.apache.http.HttpEntity
+import org.apache.http.HttpResponse
+import org.apache.http.HttpStatus
+import org.apache.http.NameValuePair
+import org.apache.http.auth.AuthScope
+import org.apache.http.auth.UsernamePasswordCredentials
+import org.apache.http.client.HttpClient
```

Comment:
why new client? What is the difference?

Reply:
More http features covered by plugin itself, more bug fixed etc. Is is easier to maintain because of up to date docs.